assets/js/country_support.js

```diff
@@ -87,7 +93,8 @@ function loadCountrySupportTable(elem, fetch) {
 
       tbody.removeChild(templateRow);
       elem.hidden = false;
-    });
+    })
+    .catch(() => {});
```

Comment:
I'm not sure I love just silently failing here, though it does technically match the current behavior, aside from the console log.

A better implementation might include some messaging to the user that we couldn't retrieve the list of supported countries.

Reply:
Yes +1 would love to have some sort of error state, would it be worth us Google Translate-ing something real quick and adding a USDWS alert "sorry try again later"?

Reply:
Gave that a quick shot here: https://github.com/18F/identity-site/pull/930